pkgs/by-name/de/deej/package.nix

```
@@ -0,0 +1,48 @@
+{ lib,
+  buildGoModule,
+  fetchFromGitHub,
+  pkg-config,
+  libappindicator-gtk3,
+  webkitgtk
+}:
+
+buildGoModule rec {
+  pname = "deej";
+  version = "0.9.10";
+  rev = "7567f83f2773e40cb950101d0784c96b1e566509";
+
+  src = fetchFromGitHub {
+    owner = "omriharel";
+    repo = "deej";
+    rev = rev;
```

```suggestion
    rev = "v${version}";
```

you can use git tags instead.